daemon/policy.go

```diff
@@ -282,6 +308,45 @@ func (d *Daemon) PolicyAdd(rules policyAPI.Rules, opts *AddOptions) (uint64, err
 	return rev, nil
 }
 
+// ReactToRuleUpdates waits until wg is complete to do the following
+// * regenerate all endpoints in epsToRegen
+// * bump the policy revision of all endpoints not in epsToRegen, but which are
+//   in allEps, to revision rev.
+func (d *Daemon) ReactToRuleUpdates(wg *sync.WaitGroup, allEps []policy.IdentityConsumer, epsToRegen *policy.IDSet, rev uint64) {
```

Comment:
Does this function need to be in the daemon package? Given you have the owner in the policy package, can we just keep it in the policy package?

Reply:
This would introduce an import cycle because `pkg/endpoint` depends on `pkg/policy`, and if I included `pkg/endpointmanager` within this function and added it to `pkg/policy`, `pkg/policy` would depend on `pkg/endpoint`. I figured this was a logical place to put this because a daemon is a 'connector' between endpointmanager, policy repository, etc.